src/runtime/components/forms/Checkbox.vue

```diff
@@ -12,7 +12,7 @@
         :indeterminate="indeterminate"
         type="checkbox"
         class="form-checkbox"
-        :class="[ui.base, ui.rounded, ui.custom]"
```

Comment:
I Noticed `custom` not exits in config or props so i added it config ?

Reply:
Just removed this in https://github.com/nuxtlabs/ui/commit/d84f3b1c6d670671160738f19aaf05d9e2c40b5f